doc/manual/install-workstation.rst

```diff
@@ -19,22 +19,25 @@ Command line submit client
 DOMjudge comes with a command line submit client which makes it really
 convenient for teams to submit their solutions to DOMjudge.
 
-In order to build the submit client, you need libcURL, libJSONcpp and
-optionally libmagic. To install this on Debian-like distributions::
+In order to use the submit client, you need python 3, the python requests 
+library and optionally the python magic library. To install this on
+Debian-like distributions::
 
-  sudo apt install libcurl4-gnutls-dev libjsoncpp-dev libmagic-dev
+  sudo apt install python3 python3-requests python3-magic
 
 Or on RedHat/CentOS/Fedora::
 
-  sudo yum install libcurl-devel jsoncpp-devel file-devel
+  sudo yum install python3 python3-requests python3-magic
 
-Then run (adapt the URL to your environment; look at the Config check
-in the admin web interface if unsure)::
+You can now copy this client from ``submit/submit`` to the workstations.
 
-  ./configure --enable-static-linking --with-baseurl="https://yourhost.example.edu/domjudge"
-  make submitclient
+To configure the baseurl used for the submit client you have four options:
 
-You can now copy this client from ``submit/submit`` to the workstations.
+* Let teams pass it using the ``--url`` argument.
+* Set it as an environment variable called ``SUBMITBASEURL``.
+* Use the ``submit_wrapper.sh`` script and set ``SUBMITBASEURL`` there.
```

Comment:
This is no longer a useful thing to have since if you can edit the wrapper you can edit the client itself. git rm submit_wrapper and change "four" in suggested text above to "three"